pkg/operation/shoot/shoot.go

```diff
@@ -171,6 +171,7 @@ func (s *Shoot) GetK8SNetworks() *gardencorev1alpha1.K8SNetworks {
 		return &s.Info.Spec.Cloud.OpenStack.Networks.K8SNetworks
 	case gardenv1beta1.CloudProviderAlicloud:
 		return &s.Info.Spec.Cloud.Alicloud.Networks.K8SNetworks
+<<<<<<< HEAD
```

Comment:
leftover?

Reply:
Yes, I stumbled across that one too. :-)